services/postgresql-docker.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "postgresql-docker",
+  "description": "PostgreSQL running in a Docker container (postgresql)",
+  "dependencies": {
+    "provides": [ "postgresql", "database" ],
```

Comment:
I am planning on adding a non-Docker `postgresql-server` service. This one is a bit tricky. The package and project name are postgresql and PostgreSQL, respectively. I have no idea why they used the shorter name for the image.